Document/0x05b-Basic-Security_Testing.md

```diff
@@ -115,13 +115,36 @@ Nevertheless several hardware characteristics can be emulated, like GPS<sup>[6]<
 -- TODO : Cf testing methodologies from CEH, ... : map attack surface (Local and Remote) through Passive and Active Reconnaissance, Scanning, Gaining Access, Maintaining Access, Covering Tracks. As this is generic and common to iOS, may be part of the parent chapter --
 
 ### Static Analysis
--- TODO : Description, when it comes compared to dynamic analysis and why, what it can bring --
+Static analysis is the act of looking into App components, source code and other resources without actually executing it. This test is focused on finding misconfigured or unprotected Android IPC components as well as finding programming mistakes such as misuse of cryptography routines, find libraries with known vulnerabilities and even dynamic code loading routines.
+
+When we are performing static analysis of applications, several tools can be used. The most commonly used are Static Analyzers. The most useful ones are Androbugs, Linkedin's QARK, Android Lint and the paid HP's Fortify. However there are a plethora of static code analyzers that can be used.
+
+Some Static Analyzers rely on the availability of the source code while others take the compiled apk as input.
+It is important to keep in mind that while static analyzers can help us to focus attention on potential problems, they may not be able to find all the problems by itself, go through each finding carefully and try to understand what the app is doing to improve your chances of finding vulnerabilities.
```

Comment:
I would add sth like this, as this is a common problem, especially with fortify ;-) 

Import to note is to configure the static analyser properly in order to reduce the likelihood of false positives and maybe only select several vulnerability categories in the scan. The results generated by static analysers can otherwise be overwhelming and the effort can become counterproductive if a 1.000 PDF report need to be manually  investigated.